File: java/src/test/java/HelloWorldShould.java
import org.junit.jupiter.api.Test;

import static org.junit.jupiter.api.Assertions.assertEquals;

public class HelloWorldShould {

    @Test
    public void returnWellKnownPhrase() {
        var helloWorld = new HelloWorld();

        var actual = helloWorld.getGreeting();

        assertEquals("Hello, World!", actual);
    }
}

File: c-sharp/Kata.Tests/HelloWorldShould.cs
namespace Kata.Tests;

public class HelloWorldShould
{
    [Test]
    public void ReturnWellKnownPhrase()
    {
        var helloWorld = new HelloWorld();

        var actual = helloWorld.GetGreeting();

        Assert.That(actual, Is.EqualTo("Hello, World!"));
    }
}
